packages/audio-block/src/AudioBlock.tsx

```diff
@@ -186,32 +198,46 @@ export const AudioBlock = ({ appBridge }: BlockProps) => {
             )}
             <div className="tw-flex tw-gap-4 tw-justify-between tw-w-full">
                 <div className="tw-flex-1">
-                    <div data-test-id="audio-block-title">
+                    {!isEditing ? (
+                        <>
+                            {hasRichTextValue(title) && (
+                                <div data-test-id="block-title-html" dangerouslySetInnerHTML={{ __html: htmlTitle }} />
+                            )}
+                        </>
+                    ) : (
                         <RichTextEditor
                             designTokens={designTokens}
                             border={false}
                             onBlur={saveTitle}
-                            placeholder={isEditing ? 'Asset name' : undefined}
-                            readonly={!isEditing}
+                            placeholder="Asset name"
                             value={title ?? DEFAULT_CONTENT_TITLE}
                             plugins={customTitlePlugins}
+                            updateValueOnChange
                         />
-                    </div>
-                    <div data-test-id="audio-block-description">
+                    )}
+                    {!isEditing ? (
```

Comment:
would do it the other way around: `{isEditing ? <RichTextEditor/> : html}`. Anyway it looks now much clearer!

Reply:
💯 adjusted